### PR TITLE
Minor cleanups and refactoring

### PR DESCRIPTION
**Pull request by sthapa**

### CODE REVIEW

File: test/DBServer.cpp [hunk 1]
```diff
@@ -212,7 +210,7 @@ int main(){
 		std::cout << "Installing federation role" << std::endl;
 		auto res = runCommand("kubectl",
 		                      {"apply", "-f",
-		                       "https://raw.githubusercontent.com/slateci/federation-controller/main/resources/installation/federation-role.yaml"});
+							   "https://raw.githubusercontent.com/slateci/federation-controller/main/resources/installation/federation-role.yaml"});
```
Comment (cnweaver) [inline test/DBServer.cpp:213]: (Alignment should use spaces, even when leading indentation is tabs.)

Comment (sthapa) [reply test/DBServer.cpp:213]: Should be fixed

**Comment by adamhgriffith-uofu** [comment]
We can ignore the trivy scan failure. That thing is just finicky.

**Comment by adamhgriffith-uofu** [comment]
With the formatting issues addressed I will plan on merging with *develop* this afternoon if there are no objections.

**Review by adamhgriffith-uofu — APPROVED** [review]

**Pull request merged**